ignition/modules/rewardmocktokens.ts

```diff
@@ -0,0 +1,6 @@
+import { buildModule } from "@nomicfoundation/hardhat-ignition/modules";
+export default buildModule("RewardMockTokens", (m) => {
+    const rewardToken = m.contract("ERC20MockToken", ["Reward Token", "RTK", 18]);
+    m.call(rewardToken, "mint", [process.env.WALLET_ADDRESS as string, "1000000000000000000000000"])
```

Comment:
Modules should only be used for deployment and for intermediate transactions or setups involved with deployment. Minting should be done separately in a script or a task.